linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/js/router_controller.js

```diff
@@ -19,6 +80,7 @@ var RouterController = (function () {
     var routerContainerEls = initializeRouterContainers(selectedRouter, routers, $parentContainer, templates.container);
 
     var routerSummaries = [];
+    var routerGraphs = [];
```

Comment:
now that each listener registers itself with the metrics collector, I don't think we need to keep track of all these any more